prime-router/src/main/kotlin/Report.kt

```diff
@@ -200,6 +200,22 @@ class Report {
     }
 */
 
+    constructor(
+        schema: Schema,
+        values: Map<String, List<String>>,
+        source: Source,
+        destination: OrganizationService? = null,
+        bodyFormat: Format? = null,
+    ) {
+        this.id = UUID.randomUUID()
+        this.schema = schema
+        this.sources = listOf(source)
+        this.bodyFormat = bodyFormat ?: destination?.format ?: Format.INTERNAL
+        this.destination = destination
+        this.createdDateTime = OffsetDateTime.now()
+        this.table = createTable(values)
+    }
+
     private constructor(
```

Comment:
Reports can now be created with a map of elements, where the string key is the column name, and the list is the column values

Reply:
Important:   if this constructor is ever used to create a new report from an existing Report.kt object, then you need to make sure to pass the `itemLineage` from the parent to the child.  Its like mitochondrial dna.

I don't think that's the case here, nor does it appear that would be a use case for this constructor in the future, but worth thinking about.